### PR TITLE
Refactor(base-url): update base url

### PR DESCRIPTION
**Pull request by Jeyob**
Move base url version before resource to keep versioning aligned

### CODE REVIEW

File: app/models/account.py [hunk 1]
```diff
@@ -32,28 +32,28 @@ class Account(db.Model):
...
                  email_consent,
                  accepted_tc):
-        self.firstname = firstname
-        self.lastname = lastname
+        self.contact_firstname = contact_firstname
+        self.contact_lastname = contact_lastname
         self.email = email
         self.hashed_password = bcrypt.generate_password_hash(password).decode('utf-8')
-        self.telephone_mobile = telephone_mobile
+        self.contact_phone = contact_phone
```
Comment (doohoo) [inline app/models/account.py:46]: Perhaps we also need an additional `contact_email`, right?

Comment (Jeyob) [reply app/models/account.py:46]: FUUUUDGE...missed it...ohh good catch!

Comment (Jeyob) [reply app/models/account.py]: Updated 💥

**Comment by Jeyob** [comment]
but will we want that?

if all we have is one master account with possibly many companies attached to it. Then each company may have a `contact_email` I suppose?

**Comment by doohoo** [comment]
Not 100% sure what we said - but I think we wanted to have a "specific" `contact_email`, that potentially could go to a "very important person" in the company ie. to the same person as the other contact fields (name & phone number).

And yes you are correct, each company _could_ have different contact details. I think we said it should default to the master accounts contact details but _could_ be overridden when a company is created.

**Pull request merged**